### ISSUE TITLE
Windows: drag-and-droppable batch files like for pngquant?

### ISSUE THREAD

**Issue by JobLeonard** [ImageOptim/gifski#37]
So recently I had the luck that I could help out a scientist friend who had gigantic, mostly-empty PNGs of his data with only a handful of colours, by using pngquant to *vastly* reduce the PNG size without any visible loss.

To my very happy surprise, it came with a drag-and-drop batch file for automatically reducing a large stack of images:

```bat
@echo off

set path=%~d0%~p0

:start

"%path%pngquant.exe" --force --verbose --quality=45-85 %1
"%path%pngquant.exe" --force --verbose --ordered --speed=1 --quality=50-90 %1

shift
if NOT x%1==x goto start
```

I was wondering: could we add the same for the Windows build of gifski? Or maybe even a version that includes ffmpeg to make converting a bunch of videos as simple as dragging/dropping it on the batch file?

**Comment by kornelski** [comment]
In theory yes. In practice, I'm not a Windows programmer, and whenever I try to do something on Windows it makes my cry, so there's zero chance it will come from me.

**Comment by JobLeonard** [comment]
Not a Windows dev either but stuck with it for now.

If you give me some sensible parameters to pass to gifski I could try working out the bat file!

I can also try to make an ffmpeg-one, put it in a sub-folder together with an html file titled `[IMPORTANT] Store ffmpeg.exe in this folder.html` which contains this:

```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>ffmpeg required</title>
  </head>
  <body style='text-align: center;font-weight: bold;'>
    <p>The drag and drop batch files only work if there is an ffmpeg.exe binary in the same folder. You can download one from:</p>
    <p><a href='https://ffmpeg.org/'>https://ffmpeg.org/</a></p>
  </body>
</html>
```

Would be about as basic yet still helpful as the instrutions can get, no?

**Comment by kornelski** [comment]
There's nothing to configure in gifski. You need to specify `-o output-path` and PNG frames.

**Comment by JobLeonard** [comment]
BTW, since generating the downloadable releases doesn't seem to be part of the repo, should I just copy the `.bat` file here once I figured it out?

**Comment by kornelski** [comment]
Yes, please drop files in `win/` folder, and I'll copy them to the package.

**Comment by majkinetor** [comment]
@JobLeonard 

FYI, basically anybody can create that kind of frontend easily in a less then hour with external tool/dsl such as AutoHotkey

**Comment by DumbJoe** [comment]
> FYI, basically anybody can create that kind of frontend easily in a less then hour with external tool/dsl such as AutoHotkey

Why dont u do it then if its so easy and quick?

**Comment by JobLeonard** [comment]
(for the record, I switched back to Linux so I haven't been able to finish this small task (and obviously also completely forgot about it, my bad))

**Comment by Droyk** [comment]
Hey @JobLeonard, I know you might be busy but can you please tell us your progress so far?

**Comment by JobLeonard** [comment]
Well...

> (for the record, **I switched back to Linux so I haven't been able to finish this small task** (and obviously also completely forgot about it, my bad))

**Comment by Droyk** [comment]
still the same ehh.. well hope you find the time someday to finish it.
